wiki/Tournaments/#GO/3/en.md

```diff
@@ -0,0 +1,159 @@
+5. There will be no warmups, so make sure you are prepared before your matches!
+6. Participants in the lobby will `!roll`. The picks are then done in descending order of the `!roll` scores.
+7. Participants will play 8 maps, each of them picking twice in the same order.
+8. After each map, the players are awarded points depending on their lobby ranking on the map:
+   - 1st place - 4 points
+   - 2nd place - 2 points
+   - 3rd place - 1 points
+   - 4th place - 0 points
+9. Only the 2 players with the most points will advance to the next stage. In case of a point tie, the Tiebreaker will be played in 1v1 between the advancing players.
+10. If any participants experienced a disconnection in the middle of play from Bancho, the player from the match will be awarded no points for the pick. The players are then given 5 minutes after the disconnection to contact their referee via osu! or Discord before being counted a loss.
+
+#### Rounds
+
+1. The match lobbies will be created around **5 - 10 minutes** ahead of the schedule.
+2. If a player fails to join the lobby past 10 minutes of his scheduled match, we will declare the match a forfeit.
+3. There will be no warmups, so make sure you are prepared before your matches!
+4. Each player will `!roll`. The highest roll decides the pick and ban order (first to ban will be the last to pick).
+5. In case of a tie, the Tiebreaker will be played. Losers in the Winners bracket will go to the Losers bracket.
+6. If any participants experienced a disconnection in the middle of play from Bancho, a rematch of that same map may be played at the referee's discretion. The players are given 5 minutes after a disconnection to contact their referee via osu! or Discord before being counted a loss.
+7. Rescheduling is allowed, however we ask of you to contact your opponent by yourself and then send us a proof that both of you are having an aggrement to reschedule. You can do so by posting a message on the `#rescheduling` text-channel on [#GO Discord server](https://discord.gg/trnFByC) with both you and your opponent acknowledging it. The deadline for rescheduling is on the Thursday 7PM (UTC) of the upcoming week of your match.
+
+### Behaviour Rules
+
+1. Every [osu! Rules](/wiki/Rules) applies.
+2. *Toxic* behaviour during matches will be highly reprimanded and may result in an **auto-loss, kick, or blacklist**.
+3. We will not tolerate any ounce of **cheating or attempts to gain an unfair advantage over the other players.**
+4. Please note that **only** the [#Game\_Osu (EN)](https://www.twitch.tv/gameosu), [#Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr), and [Sardoche's](https://www.twitch.tv/sardoche) Twitch channels are allowed to stream the tournament. **We do not allow restreams** unless given permission and will ask for **takedowns** in case of **unauthorized uploads.**However, players are allowed to stream their matches.
```

Comment:
```suggestion
4. Please note that **only** the [#Game\_Osu (EN)](https://www.twitch.tv/gameosu), [#Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr), and [Sardoche's](https://www.twitch.tv/sardoche) Twitch channels are allowed to stream the tournament. **We do not allow restreams** unless given permission and will ask for **takedowns** in case of **unauthorized uploads.** However, players are allowed to stream their matches.
```